src/components/capture/details/index.tsx

```diff
@@ -31,31 +34,47 @@ const LIVE_SPECS_QUERY = `
 `;
 
 function CaptureDetails() {
-    const intl = useIntl();
-    useTitle(
-        intl.formatMessage({
-            id: 'browserTitle.captureDetails',
-        })
-    );
+    useBrowserTitle('browserTitle.captureDetails');
 
+    const navigate = useNavigate();
     const [searchParams] = useSearchParams();
     const pubID = searchParams.get(routeDetails.capture.details.params.pubID);
+    if (!pubID) navigate(routeDetails.dashboard.path);
 
     // Supabase stuff
-    const liveSpecQuery = useQuery<LiveSpecs>(
+    const liveSpecQuery = useQuery<LiveSpecQuery>(
         TABLES.LIVE_SPECS,
         {
             columns: LIVE_SPECS_QUERY,
-            filter: (query) =>
-                query.eq('last_pub_id', pubID ? pubID : '--missing-pub_id--'),
+            filter: (query) => query.eq('last_pub_id', pubID as string),
```

Comment:
This is annoying but TS can't tell this function would only ever be called if there is a pub id.